partials/schema-prop.html

```diff
@@ -149,18 +149,19 @@
 
         {% if prop.type() === 'array' %}
           {% set arrayItemsProps = (prop.items().properties() if prop.items() else null) %}
```

Comment:
issue was that `prop.items().properties()` is always returning object, empty or not, so this `and not arrayItemsProps` could never work